fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/util/Pair.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Pair.java
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2015-2021 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.apple.foundationdb.record.util;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Utility class for a pair of objects with different types.
+ *
+ * @param <K> The type of left object
+ * @param <V> The type of right object
+ */
+public final class Pair<K, V> {
```

Comment:
Can't we use `org.apache.commons.lang3.tuple.Pair` as elsewhere in record layer?

Reply:
Fixed it. And also opened an issue and added release note for this change.

Reply:
Also added some javadoc comment for ```LuceneAnalyzerFactory.getQueryAnalyzer```.